scripts/debug_run.py

```diff
@@ -79,9 +100,22 @@ def grab_graphs(args):
       fd.write(report)
 
 
+def create_temp_folder():
+  seq = -1
+  while True:
+    dir_name = 'debug_run-{}-{}'.format(socket.gethostname(), getpass.getuser())
+    if seq >= 0:
+      dir_name += '-{}'.format(seq)
```

Comment:
this would make `dirname`, `dirname-0`, `dirname-0-1`, ... `dirname-0-1-...-n` right?

Reply:
Yes, we try not append 0 if not necessary.

Reply:
yes but why not `dirname`, `dirname-1`, `dirname-2`, .. , `dirname-10` ?

Reply:
This is just generating a unique name. There is no special meaning in the numbers.
No, this does not generate `dirname`, `dirname-0`, `dirname-0-1`, ...
But `dirname`, `dirname-0`, `dirname-1`, ...
Why is that important?

Reply:
oh bc you're resetting dirname every time. nvm it's not important.